<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMainWindow</class>
 <widget class="QMainWindow" name="UserMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="homePage">
       <property name="styleSheet">
        <string notr="true">background-color: #121212; color: #fff;</string>
       </property>
       <widget class="QLabel" name="homeTitle">
        <property name="geometry">
         <rect>
          <x>511</x>
          <y>10</y>
          <width>201</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:22px; font-weight:bold;</string>
        </property>
        <property name="text">
         <string>üéµ Home</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QListWidget" name="homeSongsList">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>88</y>
          <width>1161</width>
          <height>591</height>
         </rect>
        </property>
       </widget>
       <widget class="QFrame" name="bottomNav">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>687</y>
          <width>1156</width>
          <height>80</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
        QFrame { background-color: #1e1e1e; }
        QPushButton { 
            background: none; 
            color: #eee; 
            font-size: 24px; 
            border-radius: 10px; 
            min-width:50px; min-height:50px; max-width:50px; max-height:50px;
        }
        QPushButton:checked { background-color: #3a3a3a; }
        QPushButton:hover { background-color: #2c2c2c; }
    </string>
        </property>
        <widget class="QPushButton" name="homeBtn">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>20</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="text">
          <string>üè†</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="searchNavBtn">
         <property name="geometry">
          <rect>
           <x>550</x>
           <y>20</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="text">
          <string>üîç</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="profileBtn">
         <property name="geometry">
          <rect>
           <x>670</x>
           <y>20</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="text">
          <string>üë§</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="logoutBtn">
         <property name="geometry">
          <rect>
           <x>1100</x>
           <y>20</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background:red; color:white; font-weight:bold; border-radius:10px; min-width:50px; min-height:50px; max-width:50px; max-height:50px;</string>
         </property>
         <property name="text">
          <string>‚èª</string>
         </property>
        </widget>
        <widget class="QPushButton" name="libraryBtn">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>20</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="text">
          <string>üéµ</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>401</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:22px; font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Welcome Back, {Username}!</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:18px;;</string>
        </property>
        <property name="text">
         <string>Top 5 Artists of the Week!</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>561</width>
          <height>511</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#FFFFFF ; color: #000000;              border: 10px; solid #FFFFFF;</string>
        </property>
        <property name="rowCount">
         <number>5</number>
        </property>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>Artist ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Artist Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Weekly Listeners</string>
         </property>
        </column>
       </widget>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>150</y>
          <width>561</width>
          <height>509</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#FFFFFF ; color: #000000;              border: 10px; solid #FFFFFF;</string>
        </property>
        <column>
         <property name="text">
          <string>Artist Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Song Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Song Duration</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Play Song</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>110</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:18px;;</string>
        </property>
        <property name="text">
         <string>Your Listening History: </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="libraryPage">
       <property name="styleSheet">
        <string notr="true">background-color: #121212; color: #fff;</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="libraryTitle">
          <property name="styleSheet">
           <string notr="true">font-size:22px; font-weight:bold;</string>
          </property>
          <property name="text">
           <string>üìö Library</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="libraryList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchPage">
       <property name="styleSheet">
        <string notr="true">background-color: #121212; color: #fff;</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="searchTitle">
          <property name="styleSheet">
           <string notr="true">font-size:22px; font-weight:bold;</string>
          </property>
          <property name="text">
           <string>üîç Search</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLineEdit" name="searchInput">
            <property name="placeholderText">
             <string>Search for songs or artists...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchBtn">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="searchResults"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="profilePage">
       <property name="styleSheet">
        <string notr="true">background-color: #121212; color: #fff;</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="profileTitle">
          <property name="styleSheet">
           <string notr="true">font-size:22px; font-weight:bold;</string>
          </property>
          <property name="text">
           <string>üë§ Profile</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="themeToggleBtn">
          <property name="text">
           <string>Switch Light/Dark Theme</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
